### ISSUE TITLE
Prices for fertilizers

### ISSUE THREAD

**Issue by jaspercacbay** [exnil/crop_planner#3]
Is it just me or is it not taken into account?

**Comment by exnil** [comment]
Fertilizer price is currently not taken into account.

I haven't planned on adding this yet, I would need to work out a way for users to select whether they are buying the fertilizer or crafting it, which would determine the cost. The add-crop menu is already too cluttered for this at the moment.

**Comment by forlornhedgehog** [comment]
Agreed add-crop menu too cluttered for more items. Perhaps it can be assumed users are buying the fertilizer, as buying seeds (as opposed to using seed maker) is also already assumed.

**Comment by exnil** [comment]
Good point, thanks.
I'd also assume it's more common for players to buy fertilizer rather than to craft it.
